keycloak/keycloak-operator
# KEYCLOAK-12669 Add PodDisruptionBudget support

## JIRA ID

[KEYCLOAK-12669](https://issues.redhat.com/browse/KEYCLOAK-12669)

## Additional Information

Discussions about the topic can be found [here](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/keycloak-dev/A8F-sERqCGE/TmpO6p2MBwAJ)

## Verification Steps
1. Deploy the operator. You can use this image: `quay.io/wei_lee/keycloak-operator:dev`
2. Create a Keycloak CR. You can use the example one in this PR. At this point, there should be no PDB created yet as there is only 1 instance of Keycloak is provisioned.
3. Increase the number of Keycloak instances and you should see a PDB is created for Keycloak
4. Decrease the number of Keycloak to 1 again and you should see the PDB is removed.

## Checklist:
